botorch/utils/sampling.py

```diff
@@ -379,7 +379,7 @@ def __init__(
 
         if equality_constraints is not None:
             self.C, self.d = equality_constraints
-            U, S, V = torch.svd(self.C, some=False)
+            U, S, V = torch.linalg.svd(self.C)
```

Comment:
`torch.linalg.svd` is different from `torch.svd` to be NumPy compatible. The third returned tensor is actual now is the conjugate transpose of `V`.
```suggestion
            U, S, Vh = torch.linalg.svd(self.C)
            V = Vh.conj().transpose(-2, -1)
```